uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -481,6 +483,94 @@ public void testCreateZone_ZoneAlreadyExists() {
         assertResponse(expectedResponse, getResponseAlreadyExist.getBody());
     }
 
+    @Test
+    public void testImportZone() {
```

Comment:
This test should verify the generate token using the adminClientSecret after importing.

Reply:
Shouldn't that be covered under functional test ? Usually Integration test only covers the the integration of the code at different layers. In the code that is being tested with this integration test ,  adminsecret is not being generated . I think testing for adminClientSecretwill be beyond the scope of this integration test . Please suggest.